swri_transform_util/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ find_package(tf2 REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(yaml_cpp_vendor REQUIRED)
-find_package(yaml-cpp REQUIRED)
```

Comment:
This find_package is needed, see pattern used in other ros packages:
https://github.com/ros2/rviz/blob/rolling/rviz_common/CMakeLists.txt#L56-L57
https://github.com/ros-perception/image_common/blob/rolling/camera_calibration_parsers/CMakeLists.txt#L19-L20